site/source/user-manual/backups/backup-setup/backup-mac.rst

```diff
@@ -1,8 +1,10 @@
-.. _cifs-mac:
+.. _backup-mac:
```

Comment:
This Mac page looks totally different than Linux and Windows. The latter have sections for "Create LAN Shared Folder" and "Connect Embassy". Each platform page should follow the same format.